Automattic/wp-calypso
# Plans: new Jetpack Product card UI

#### Changes proposed in this Pull Request

* Introduce the new simplified version of the `JetpackProductCard` component.

#### Notes

There are still things to do, but I prefer to get this merged so others can start contributing as well. The things we are missing are:

* Make sure the copy is up-to-date.
* Use new icons for features.
* Make single product icons darker blue (`#2271B1`).

#### Caveats

* This PR doesn't deal with the layout of the page so don't expect to see the 3 columns yet. The size of the cards might be affected by this.
* We still have options cards instead of daily ones (you can navigate to the Options page) because this PR won't deal with that.
* Upsell are still in place. We will deal with them in another PR.

#### Testing instructions

* Run this PR locally (run both Calypsos).
* Replace line 17 of `client/my-sites/plans-v2/plans-column/index.tsx` with `import ProductCard from '../product-card-alt';`. We want to use the alternative version of the `ProductCard` component.
* Replace line 13 of `client/my-sites/plans-v2/products-column/index.tsx` with `import ProductCard from '../product-card-alt';`. We want to use the alternative version of the `ProductCard` component.
* Visit `http://jetpack.cloud.localhost:3001/pricing`.
* Make sure the product cards look as per design (remember there is still work to do).
* Visit `http://calypso.localhost:3000/plans/:site`.

**Take your time to verify that nothing from the previous version of the cards has been affected. Everything should work and look the same if you undo the changes made to the two files mentioned above.**

Fixes 1196341175636977-as-1198071178569629

#### Demo
##### cloud.jetpack.com/pricing
![image](https://user-images.githubusercontent.com/3418513/95603178-4e188280-0a2c-11eb-8216-8d12aab48a2f.png)

##### wordpress.com/plans
![image](https://user-images.githubusercontent.com/3418513/95603123-3b05b280-0a2c-11eb-8f84-264f40b8aa94.png)

##### wordpress.com/plans
![image](https://user-images.githubusercontent.com/3418513/95603058-26c1b580-0a2c-11eb-92ba-90c9afd40886.png)
